### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
-# AJAX Control Toolkit (maintained by [DevExpress](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressHome.aspx))
+# ASP.NET AJAX Control Toolkit
```
Comment (Abadzhev) [inline README.md:1]:
```suggestion
# ASP.NET AJAX Control Toolkit (for ASP.NET 4.x)
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
 [![Installer](https://img.shields.io/badge/get%20installer-20.1.0-orange.svg)](https://go.devexpress.com/AjaxControlToolkit_Website_Download.aspx)
 
-The ASP.NET AJAX Control Toolkit contains a rich set of controls that you can use to build highly responsive and interactive AJAX-enabled Web applications. AjaxControlToolkit contains more than 40 controls, including the AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark controls. Using AjaxControlToolkit, you can build AJAX-enabled ASP.NET Web Forms applications by dragging-and-dropping Toolkit controls from the Visual Studio Toolbox onto a Web Forms page.
-
-**v20.1.0 is now available**! [Get the new installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx).
+The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains more than 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark. Use these controls to create highly responsive and interactive web applications.
```
Comment (Abadzhev) [inline README.md:6]:
```suggestion
The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
 
 ![logo](https://github.com/DevExpress/AjaxControlToolkit/raw/wiki-static-resources/readme-banner.png)
 
-If you've used the toolkit in the past, you also know that for quite some time, the project was poorly supported, with an ever growing list of critical issues. Seeing the demand for such a library and considering its current state, we approached Microsoft and suggested that we help bring the Toolkit back into a dependable and usable state. Needless to say, Microsoft agreed. The good news is that since we took on this responsibility, we've managed to clean up the codebase, introduced improved Visual Studio support, and fixed nearly 900 bugs.
+Download the DevExpress Edition of the AJAX Control Toolkit today, evaluate it in your project and let us know what you think.
```
Comment (Abadzhev) [inline README.md:10]:
```suggestion
Use the following link to download AJAX Control Toolkit:  
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
 
 ![logo](https://github.com/DevExpress/AjaxControlToolkit/raw/wiki-static-resources/readme-banner.png)
 
-If you've used the toolkit in the past, you also know that for quite some time, the project was poorly supported, with an ever growing list of critical issues. Seeing the demand for such a library and considering its current state, we approached Microsoft and suggested that we help bring the Toolkit back into a dependable and usable state. Needless to say, Microsoft agreed. The good news is that since we took on this responsibility, we've managed to clean up the codebase, introduced improved Visual Studio support, and fixed nearly 900 bugs.
+Download the DevExpress Edition of the AJAX Control Toolkit today, evaluate it in your project and let us know what you think.  
+[Get the new installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx)
```
Comment (Abadzhev) [inline README.md:11]:
```suggestion
[Download the Installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx)
```

File: README.md [hunk 5]
```diff
@@ -1,29 +1,27 @@
...
 
-The ASP.NET AJAX Control Toolkit contains a rich set of controls that you can use to build highly responsive and interactive AJAX-enabled Web applications. AjaxControlToolkit contains more than 40 controls, including the AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark controls. Using AjaxControlToolkit, you can build AJAX-enabled ASP.NET Web Forms applications by dragging-and-dropping Toolkit controls from the Visual Studio Toolbox onto a Web Forms page.
-
-**v20.1.0 is now available**! [Get the new installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx).
+The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains more than 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark. Use these controls to create highly responsive and interactive web applications.
 
 ![logo](https://github.com/DevExpress/AjaxControlToolkit/raw/wiki-static-resources/readme-banner.png)
```
Comment (Abadzhev) [inline README.md:8]: Let's remove that image. 

```suggestion

```

File: README.md [hunk 6]
```diff
@@ -1,29 +1,26 @@
...
 [![Installer](https://img.shields.io/badge/get%20installer-20.1.0-orange.svg)](https://go.devexpress.com/AjaxControlToolkit_Website_Download.aspx)
 
-The ASP.NET AJAX Control Toolkit contains a rich set of controls that you can use to build highly responsive and interactive AJAX-enabled Web applications. AjaxControlToolkit contains more than 40 controls, including the AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark controls. Using AjaxControlToolkit, you can build AJAX-enabled ASP.NET Web Forms applications by dragging-and-dropping Toolkit controls from the Visual Studio Toolbox onto a Web Forms page.
-
-**v20.1.0 is now available**! [Get the new installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx).
+The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.
```
Comment (dirk-pieterse) [inline README.md:6]:
```suggestion
The ASP.NET AJAX Control Toolkit is an open-source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.
```

**Review by dirk-pieterse — APPROVED** [review]

**Review by romanresh — APPROVED** [review]

**Pull request merged**